src/timeline/RoomlistModel.cpp

```diff
@@ -158,6 +164,13 @@ RoomlistModel::data(const QModelIndex &index, int role) const
                                 return false;
                         case Roles::Tags:
                                 return QStringList();
+                        case Roles::IsDirect:
+                                return room.member_count == 1;
```

Comment:
Shouldn't this be 2?

Reply:
No. If it's an invite, you aren't a member yet, so it is 1.